101-container-registry-geo-replication/azuredeploy.json

```diff
@@ -0,0 +1,80 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "acrName": {
+      "type": "string",
+      "minLength": 5,
+      "maxLength": 50,
+      "metadata": {
+        "description": "Name of your Azure Container Registry"
```

Comment:
does this need to be globally unique?  If so, add a defaultValue w/ uniqueString and update the description

Reply:
I had GEN-UNIQUE in the parameters file, but have also updated to include a default value